### PR TITLE
Store color as UInt32

### PR DESCRIPTION
**Pull request by vanwagonet**
fixes a crash on assignment details

[ignore-commit-lint]

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Core/Core/Extensions/UIColorExtensions.swift [hunk 1]
```diff
@@ -47,11 +47,23 @@ extension UIColor {
...
+    public convenience init(intValue value: UInt32) {
+        self.init(
+            red: CGFloat((value & 0xff0000) >> 16) / 255,
+            green: CGFloat((value & 0x00ff00) >> 8) / 255,
+            blue: CGFloat((value & 0x0000ff) >> 0) / 255,
+            alpha: CGFloat((value & 0xff000000) >> 24) / 255
+        )
+    }
+
+    public var intValue: UInt32 {
```
Comment (armstrongnate) [inline Core/Core/Extensions/UIColorExtensions.swift]: Tests for this and the new init?

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1007629&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.33% | 93.33% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**